README.md

```diff
@@ -89,6 +89,61 @@ cd /home/YOUR_USERNAME/CMAP
 
 python train.py configs.config --experiment_name <ExperimentName> --aug_type <aug> --split <split> --num_trial <num_trial>
 ```
+### Running Hyperparameter Sweep with SLURM and Weights & Biases
```

Comment:
Please move this closer to the bottom of the README and add another markdown file you link to for "future work" (great idea of yours during today's meeting).

In the future work section, describe what the next steps are to get this working with Slurm.